integrations/other/sqlmesh.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: "Use SQLMesh to streamline pipelines"
+description: "Introduces how SQLMesh simplifies managing RisingWave streaming SQL pipelines."
+sidebarTitle: SQLMesh
+---
+
+This guide explains how to use RisingWave for streaming data and SQLMesh for managing transformations to aggregate website click events. Each event contains a `timestamp`, `event_type`, and `value`.
```

Comment:
Let's add a bit more context here about the role that SQLMesh plays in managing pipelines. For example, you can fit the descriptions from these two sections/blocks:

<img width="726" alt="image" src="https://github.com/user-attachments/assets/8095e227-c5c0-427f-a0c2-46c8c46f5eae" />
<img width="748" alt="image" src="https://github.com/user-attachments/assets/145a70d2-9322-4394-ad90-ffc758ae485d" />
And then go on to mention that we'll use a demo to show how it works.